src/Compilers/VisualBasic/Test/Semantic/FlowAnalysis/FlowTestBase.vb

```diff
@@ -140,7 +140,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests
         End Function
 
         Private Sub AdjustToken(ByRef token As SyntaxToken)
-tryAgain :
+tryAgain:
```

Comment:
@gafter @jaredpar Yes, Labels are always aligned to (Column 0)Left margin in VB. Also no spacing between the label and the colon. The resultant above is the desired format.